data/strings/strings.txt

```diff
@@ -31674,3 +31627,11 @@
     vi = Không thể hiển thị hộp thoại chọn thư mục vì không có ứng dụng phù hợp nào được cài đặt trên thiết bị của bạn. Vui lòng cài đặt ứng dụng quản lý tệp và thử lại
     zh-Hans = 由于您的设备上未安装合适的应用程序，因此无法显示文件夹选择对话框。请安装文件管理器应用程序并重试。
     zh-Hant = 無法顯示資料夾選擇對話框，因為您的裝置上沒有安裝合適的應用程式。請安裝檔案總管應用程式並重試
+
+  [choose_color]
+    tags = android
+    en = Choose Color
+
+  [edit_track]
+    tags = android
+    en = Edit Track
```

Comment:
@kavikhalique could you please open a PR with translations of "Edit Track" and "Choose Color" to other languages?

Reply:
Yeah sure i will do it shortly